components/rememes/Rememes.module.css

```
@@ -236,25 +220,22 @@
 
 .addRememeTokenList {
   position: relative;
   max-height: 40vh;
   overflow-y: scroll;
-
-  li {
-    word-wrap: break-word;
-  }
+}
+.addRememeTokenList li {
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)